weasel/_util.py

```diff
@@ -459,7 +458,7 @@ def get_git_version(
     """
     try:
         ret = run_command("git --version", capture=True)
-    except:
+    except subprocess.SubprocessError:
```

Comment:
Is this the only possible exception? Maybe just:

```suggestion
    except Exception:
```

Reply:
(for a formatting PR at least, not that this couldn't change in the future)

Reply:
Thank you!